core/src/main/java/tech/pegasys/web3signer/core/config/Config.java

```diff
@@ -37,6 +38,8 @@ public interface Config {
 
   Path getKeyConfigPath();
 
+  YAMLMapper getKeyConfigYamlMapper();
```

Comment:
Seems a reasonable approach for this isolated usage of YAMLMapper. 
If we needed other instances of YAMLMapper in the prod code, they would exist outside of this config I guess?

Reply:
correct ... as you can see the test code uses YamlMapperFactory directly for unit/integration testing. The main goal is to keep all the configurations for creating YamlMapper at one spot.